central/detection/service/service_impl_test.go

```diff
@@ -685,3 +690,23 @@ func TestFetchOptionFromRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAppliedNetpolsForDeployment(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	mockNetpolDS := networkPolicyMockStore.NewMockDataStore(mockCtrl)
+	mockNetpolDS.EXPECT().GetNetworkPolicies(gomock.Any(), gomock.Any(), gomock.Any()).Return(
+		[]*storage.NetworkPolicy{
+			{Id: uuid.NewV4().String(), Spec: &storage.NetworkPolicySpec{PodSelector: &storage.LabelSelector{MatchLabels: map[string]string{"app": "match"}}}},
+			{Id: uuid.NewV4().String(), Spec: &storage.NetworkPolicySpec{PodSelector: &storage.LabelSelector{MatchLabels: map[string]string{"nomatch": "nomatch"}}}},
+		}, nil)
+	s := serviceImpl{
+		netpols: mockNetpolDS,
+	}
+	eCtx := enricher.EnrichmentContext{Namespace: "ns", ClusterID: uuid.NewV4().String()}
+	d := storage.Deployment{Id: uuid.NewV4().String(), PodLabels: map[string]string{"app": "match"}}
+
+	actual, err := s.getAppliedNetpolsForDeployment(context.Background(), eCtx, &d)
```

Comment:
[nit] I see again the var `actual` from that pattern you mentioned last time. I usually stick to `got` as it is named in unit tests generated automatically by VSCode. Do you know maybe whether go has an unified opinion on naming this var?

Reply:
I've quickly checked the go source itself, as well as an[ article on naming conventions](https://ieftimov.com/posts/testing-in-go-naming-conventions/).
It seems that both patterns, `actual`/`expected` and `got`/`want` are used in the Go codebase itself as well as popular projects.
Maybe this is a good extension for our code guidelines. 
@rukletsov would you happen to have an informed opinion on this matter? 😄

Reply:
That is fine, I am resolving the comment, but feel free to wait on Alex'es opinion.